crates/rome_js_semantic/src/semantic_model/builder.rs

```diff
@@ -93,8 +93,8 @@ impl SemanticModelBuilder {
     }
 
     #[inline]
-    pub fn push_global(&mut self, name: impl Into<String>) {
-        self.globals_by_name.insert(name.into(), None);
+    pub fn push_global(&mut self, name: String) {
+        self.globals_by_name.insert(name, None);
```

Comment:
The previous API was more flexible. And the result doesn't change because we always need to allocate a new `String`.

I *don't think* the compiler creates a new `String` when we have `String.into::<String>()`

Reply:
I removed some `Into<>` mostly because they were not used. Do you think we need to keep them?

Reply:
That's usually the signature that you want when you pass a string to a function, because you can pass any kind of string.